index.js

```diff
@@ -178,18 +182,23 @@ function SpotifyWebHelper(opts) {
       getJSON({
         url: 'http://open.spotify.com/token'
       })
-        .then(function (res) {
-          resolve(res.t);
-        })
-        .catch(reject);
+      .then(function (res) {
+        resolve(res.t);
+      })
+      .catch(reject);
     });
   };
   this.checkForError = status => {
-    if (!status.open_graph_state) {
-      this.player.emit('error', new Error('No user logged in'));
-      return true;
-    }
     if (status.error) {
+      if (status.error.message === 'Invalid OAuth token') {
+        // TODO: It would be great to have something more precise than just the message to check.
```

Comment:
This method of error handling is acceptable, if the status objects include an error code that would be more suitable, but they don't (I don't believe they do at least).  Maybe we can add other error checks here to better handle them as well.